### ISSUE TITLE
Include protocols in redis stats

### ISSUE THREAD

**Issue by dbrgn** [coturn/coturn#108]
Would be great if the client and relay protocols would be logged to the redis stats database. We use this database to log usage information to Grafana.

**Comment by misi** [comment]
It would be great if you could you send a PR,,

**Comment by eakraly** [comment]
Hi @dbrgn !
Do you think it is still valuable for you to get this feature?

**Comment by dbrgn** [comment]
@eakraly yes, it would be interesting information for monitoring production services!
